src/garage/torch/algos/_utils.py

```diff
@@ -110,7 +110,7 @@ def compute_advantages(discount, gae_lambda, max_path_length, baselines,
     deltas = (rewards + discount * F.pad(baselines, (0, 1))[:, 1:] - baselines)
     deltas = F.pad(deltas, (0, max_path_length - 1)).unsqueeze(0).unsqueeze(0)
 
-    advantages = F.conv2d(deltas, adv_filter, stride=1).squeeze()
+    advantages = F.conv2d(deltas, adv_filter, stride=1).view(rewards.shape)
```

Comment:
i also saw an update to this in #1188. Is this a bug?

Please confer with @yonghyuc and perhaps open a separate issue/PR so that it can be backported.

Reply:
Yes, this is a bug that breaks when `rewards` has a shape of something like `(1, N)`. The expected shape of `advantages` is the same of that of `rewards`, but the `Tensor.seqeeze()` will squeeze out the the first dimension in this case.

Reply:
@yonghyuc Because this is a bug that impacts old release of garage, you need to make a backport PR for this. If you don't know how to do it, here is an example https://github.com/rlworkgroup/garage/pull/1079.

Reply:
Backport PR opened at #1197